src/LoRa.cpp

```diff
@@ -412,6 +412,11 @@ void LoRaClass::sleep()
   writeRegister(REG_OP_MODE, MODE_LONG_RANGE_MODE | MODE_SLEEP);
 }
 
+void LoRaClass::continuosMode()
```

Comment:
typo in method name ? continuosMode -> continuousMode

Reply:
Ready